app/models/container_template.rb

```diff
@@ -1,5 +1,6 @@
 class ContainerTemplate < ApplicationRecord
   include CustomAttributeMixin
+  include ManageIQ::Providers::Kubernetes::ContainerManager::EntitiesMapping
```

Comment:
The theory I've always used as a guidance is that one day a non-kubernetes containers provider might be added. (*)
On these grounds, a mapping involving kubernetes/openshift entity names probably does not belong in core.
And a core model shouldn't depend on provider code.
BUT.  It's not exactly true all our models are in core — we have ContainerManager *subclasses* inside k8s & openshift provider repos.

- So one way for "laundering" this dependence is including EntitiesMapping in Kubernetes::ContainerManager and ContainerTemplate delegating the mapping through its EMS.
- Another is admitting you don't instantiate an abstract ContainerTemplate.  Looking at the code here, it's quite kube/openshift-specific.  We could subclass a Providers::Openshift::ContainerTemplate, move code there & include EntitiesMapping there.

(*) That's just "code logic" 🖖.  @ohadlevy, the pluggability/dev experience/lifecycle considerations you're looking at are more important 👷